core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -600,8 +600,14 @@ private InputStream getBlockInStream(long blockId) throws IOException {
         throw e;
       }
       long blockStart = BlockId.getSequenceNumber(blockId) * mBlockSize;
-      return createUnderStoreBlockInStream(blockStart, getBlockSize(blockStart),
-          mStatus.getUfsPath());
+      try {
+        return createUnderStoreBlockInStream(blockStart, getBlockSize(blockStart),
+            mStatus.getUfsPath());
+      } catch (IOException e2) {
+        LOG.debug("Failed to read from UFS after failing to read from Alluxio: {}", e2);
```

Comment:
Won't this implementation only log the message since we are using `{}` instead of just tacking the exception on at the end?